src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -619,6 +616,16 @@ public void beforeCommand(CommandEnvironment env) throws AbruptExitException {
               executorService, env, remoteCache, retryScheduler, digestUtil);
     }
 
+    buildEventArtifactUploaderFactoryDelegate.init(
```

Comment:
Should we use a `buildEventArtifactUploaderFactoryDelegateAdapterFacade.init()` here instead? (just kidding ;))